testing/asserts.ts

```diff
@@ -438,6 +438,48 @@ delta "${f(delta)}" is greater than "${f(tolerance)}"`,
   );
 }
 
+// deno-lint-ignore no-explicit-any
+type AnyConstructor = new (...args: any[]) => any;
+type GetConstructorType<T extends AnyConstructor> = T extends // deno-lint-ignore no-explicit-any
+new (...args: any) => infer C ? C
+  : never;
```

Comment:
The formatting here is rather weird, but afaik this really needs `any`. I tried `unknown` but that causes issues when passing constructors with parameters.